templates/control-tower-integration.template.yaml

```diff
@@ -356,11 +356,16 @@ Resources:
             - organizations:DescribeAccount
             Resource:
               !Join ['', ['arn:aws:cloudformation:', !Ref 'AWS::Region', ':', !Ref 'AWS::AccountId', ':stackset/', '*' ]]
-          - Sid: StackSetInstanceDelete
+         - Sid: StackSetInstanceDelete
```

Comment:
I think you accidentally dropped a space from this line (which will likely anger the yaml gods)